lib/common/common/src/mmap_hashmap.rs

```diff
@@ -241,13 +244,21 @@ impl<K: Key + ?Sized, V: Sized + FromBytes + Immutable + IntoBytes + KnownLayout
     }
 
     /// Get the values associated with the `key`.
-    pub fn get(&self, key: &K) -> io::Result<Option<&[V]>> {
+    pub fn get(&self, key: &K, hw_counter: &HardwareCounterCell) -> io::Result<Option<&[V]>> {
         let Some(hash) = self.phf.get(key) else {
             return Ok(None);
         };
 
+        hw_counter
+            .payload_index_io_read_counter()
+            .incr_delta(BUCKET_OFFSET_OVERHEAD);
+
         let entry = self.get_entry(hash as usize)?;
 
+        hw_counter
+            .payload_index_io_read_counter()
+            .incr_delta(size_of_val(entry));
```

Comment:
Since it's just a `&[u8]` we can take it's length directly.

```suggestion
            .incr_delta(entry.len());
```